set.mm

```diff
@@ -273679,16 +273733,16 @@ regarding the multiplication ( ~ rngqiprnglin ) via ~ rngqiprnglinlem1
     idomdomd $p |- ( ph -> R e. Domn ) $=
       ( ccrg cdomn cidom cin df-idom eleqtrdi elin2d ) ADEBABFDEGCHIJ $.
 
+    $( An integral domain is a commutative ring with unity.  (Contributed by
+       Thierry Arnoux, 4-May-2025.)  Use ~ idomringd subproof.  (Proof
```

Comment:
"Use ~ idomringd subproof." => maybe better "Formerly subproof of ~ idomringd .", because this will not be a subproof of ~ idomringd anymore (after this PR is merged).